tests/v2/validation/rbac/rbac_namespace_move_test.go

```diff
@@ -0,0 +1,200 @@
+
+	steveStdUserclient, err := rb.standardUserClient.Steve.ProxyDownstream(rb.cluster.ID)
+	require.NoError(rb.T(), err)
+	rb.steveNonAdminClient = steveStdUserclient
+
+	_, err = rb.steveNonAdminClient.SteveType(namespaces.NamespaceSteveType).Update(projectReadOnlyNamespace, updateNS)
+	require.Error(rb.T(), err)
+	errStatus := strings.Split(err.Error(), ".")[1]
+	rgx := regexp.MustCompile(`\[(.*?)\]`)
+	errorMsg := rgx.FindStringSubmatch(errStatus)
+	assert.Equal(rb.T(), "403 Forbidden", errorMsg[1])
+}
+
+func (rb *NameSpaceMoveTestSuite) ValidateStandardUserCannotMoveANamespaceToReadOnlyProject() {
+	// Create namespace under project owner project
+	projectOwnerNamespaceName := namegen.AppendRandomString("testns-")
+	projectOwnerNamespace, err := namespaces.CreateNamespace(rb.client, projectOwnerNamespaceName+"-standard-user-project-owner-permission", "{}", map[string]string{}, map[string]string{}, rb.standardUserPO)
+	require.NoError(rb.T(), err)
+
+	// Create namespace under project read only project
+	projectReadOnlyNamespaceName := namegen.AppendRandomString("testns-")
+	projectReadOnlyNamespace, err := namespaces.CreateNamespace(rb.client, projectReadOnlyNamespaceName+"-standard-user-project-read-only-permission", "{}", map[string]string{}, map[string]string{}, rb.standardUserPR)
+	require.NoError(rb.T(), err)
+
+	// Validate user cannot move namespace from project read only to project owner role
+	steveAdminClient, err := rb.client.Steve.ProxyDownstream(rb.cluster.ID)
+	require.NoError(rb.T(), err)
+	rb.steveAdminClient = steveAdminClient
+
+	updateNS, err := getAndConvertNamespace(projectOwnerNamespace, rb.steveAdminClient)
+	require.NoError(rb.T(), err)
+	updateNS.Labels = projectReadOnlyNamespace.Labels
+	updateNS.Annotations = projectReadOnlyNamespace.Annotations
+
+	steveStdUserclient, err := rb.standardUserClient.Steve.ProxyDownstream(rb.cluster.ID)
+	require.NoError(rb.T(), err)
+	rb.steveNonAdminClient = steveStdUserclient
+
+	_, err = rb.steveNonAdminClient.SteveType(namespaces.NamespaceSteveType).Update(projectOwnerNamespace, updateNS)
+	require.Error(rb.T(), err)
+	errStatus := strings.Split(err.Error(), ".")[1]
+	rgx := regexp.MustCompile(`\[(.*?)\]`)
+	errorMsg := rgx.FindStringSubmatch(errStatus)
+	assert.Equal(rb.T(), "403 Forbidden", errorMsg[1])
+}
+
+func (rb *NameSpaceMoveTestSuite) TestNameSpaceMoveTestSuite() {
+
+	rb.Run("Create two projects with different roles", func() {
+		rb.CreateProjects()
+	})
+
+	rb.Run("Validate if Admin user can move a namespace to a Read Only project", func() {
+		rb.ValidateAdminCanMoveANamespaceWithReadOnlyProject()
+	})
+
+	rb.Run("Validate if standard user cannot move a namespace from a Read Only project", func() {
+		rb.ValidateStandardUserCannotMoveANamespaceFromReadOnlyProject()
+	})
+
+	rb.Run("Validate if standard user cannot move a namespace to a Read Only project", func() {
+		rb.ValidateStandardUserCannotMoveANamespaceToReadOnlyProject()
```

Comment:
Similar to the above This should be a ValidateProjectOwner as a standard user who is also a Cluster Owner can move the namespaces.